.happy/terraform/modules/cloudwatch-alarm/main.tf

```
@@ -105,14 +105,16 @@ resource aws_cloudwatch_log_metric_filter frontend_uncaught_error {
   pattern         = join(" ", [
     "\"com.amazon.rum.js_error_event\"",
     # Errors that occur when fetching RUM credentials. Safe to ignore because
-    # this is due to the user's network environment.
+    # this is due to the user's network environment:
+    # https://github.com/aws-observability/aws-rum-web/issues/227#issuecomment-1252712017
     "-\"CWR: Failed to retrieve Cognito identity\"",
     "-\"CWR: Failed to retrieve credentials\"",
+    "-\"CWR: Failed to retrieve Cognito OpenId token\"",
```

Could we generalize this to silence all errors containing the term `Cognito` or `credentials`? As we don't want to be alerted on failures of RUM.